### ISSUE TITLE
Deprecate old versions.

### ISSUE THREAD

**Issue by jabrena** [prosociallearnEU/cf-nodejs-client#61]
https://docs.npmjs.com/cli/deprecate

**Comment by jabrena** [comment]
npm deprecate cf-nodejs-client@"< 0.9.2" "critical bug fixed in v 0.9.2"

**Comment by jabrena** [comment]
All dependents were warned.

**Comment by jabrena** [comment]
npm deprecate cf-nodejs-client@"< 0.11.0" "critical issues fixed in v 0.11.0"

**Comment by jabrena** [comment]
``` bash
npm install cf-nodejs-client@0.10.0 --save
npm WARN deprecated cf-nodejs-client@0.10.0: critical issues fixed in v 0.11.0
helloworld@0.1.0 C:\Users\XXX\Documents\YYY\cf-nodej
s-client-docs\examples\helloWorld
└─┬ cf-nodejs-client@0.10.0
  └─┬ restler@3.4.0
    └─┬ xml2js@0.4.0
      └─┬ xmlbuilder@4.1.0
        └── lodash@3.10.1
```
